### PR TITLE
implements UUID

### PR DESCRIPTION
**Pull request by Manuel9703**

### CODE REVIEW

**Review by LucaCasella — APPROVED** [review]

File: database/migrations/2014_10_12_000000_create_users_table.php [hunk 1]
```diff
@@ -14,12 +14,9 @@
     public function up()
     {
         Schema::create('users', function (Blueprint $table) {
-            $table->id();
+            $table->uuid('id');
             $table->string('name');
             $table->string('email')->unique();
-            $table->timestamp('email_verified_at')->nullable();
```
Comment (antonio8101) [inline database/migrations/2014_10_12_000000_create_users_table.php:20]: This lines should not be removed. Please restore them

File: tests/Unit/ExampleTest.php [hunk 2]
```diff
@@ -3,6 +3,7 @@
 namespace Tests\Unit;
 
 use PHPUnit\Framework\TestCase;
+use App\Models\User;
```
Comment (antonio8101) [inline tests/Unit/ExampleTest.php:6]: Remove this line

**Review by antonio8101 — CHANGES_REQUESTED** [review]
Some minor issues..

File: tests/Unit/ExampleTest.php (hunk 2 above)
Comment (Manuel9703) [reply tests/Unit/ExampleTest.php:6]: the file with that namespace in my project does not exist...

**Review by antonio8101 — APPROVED** [review]

**Pull request merged**